sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineOperationsTests.java

```diff
@@ -104,6 +108,34 @@ protected void cleanUpResources() {
         }
     }
 
+    @Test
+    public void canCreateAndUpdateVirtualMachineWithUserData() {
+        String userDataForCreate = new String(Base64.getEncoder().encode(
+            UUID.randomUUID().toString().toUpperCase(Locale.ROOT).getBytes(StandardCharsets.UTF_8)));
```

Comment:
randomUUID cannot be created this way. The random data need to be recorded in the test record.

Use `ResourceManagerUtil.randomUuid`.

Reply:
Or just use some non-random string. If the content is not important.